src/backend/distributed/citus--7.0-9--7.0-10.sql

```diff
@@ -0,0 +1,27 @@
+/* citus-7.0-9--7.0-10 */
+
+SET search_path = 'pg_catalog';
+
+CREATE FUNCTION master_add_secondary_node(nodename text,
+                                          nodeport integer,
+                                          primaryname text,
+                                          primaryport integer,
+                                          nodecluster name default 'default',
+                                          OUT nodeid integer,
+                                          OUT groupid integer,
+                                          OUT nodename text,
+                                          OUT nodeport integer,
+                                          OUT noderack text,
+                                          OUT hasmetadata boolean,
+                                          OUT isactive bool,
+                                          OUT noderole noderole,
+                                          OUT nodecluster name)
```

Comment:
I wonder whether we should keep repeating past mistakes (i.e. not just returning a boolean here), but inconsistent signatures feels slightly worse.